endpoints/openrtb2/auction.go

```diff
@@ -2319,9 +2319,11 @@ func (deps *endpointDeps) processStoredRequests(requestJson []byte, impInfo []Im
 
 // parseImpInfo parses the request JSON and returns impression and unmarshalled imp.ext.prebid
 func parseImpInfo(requestJson []byte) (impData []ImpExtPrebidData, errs []error) {
+	// ignore the error such as '&errors.errorString{s:"Unknown value type"}'
 	if impArray, dataType, _, err := jsonparser.Get(requestJson, "imp"); err == nil && dataType == jsonparser.Array {
-		_, err = jsonparser.ArrayEach(impArray, func(imp []byte, _ jsonparser.ValueType, _ int, err error) {
-			impExtData, _, _, err := jsonparser.Get(imp, "ext", "prebid")
+		_, _ = jsonparser.ArrayEach(impArray, func(imp []byte, _ jsonparser.ValueType, _ int, _ error) {
+			// ignore possible errors if there is no ext.prebid key
```

Comment:
I think you can remove this comment

Reply:
Just checking to see what you think about removing this

Reply:
removed in 55345bfb